app/Http/Requests/BulkAssetCheckoutRequest.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
In the future, we probably don't need a form request for a one-off like this. Probably easier/better to just invoke the validator in the controller method, since there's only one validation rule.